.deepsource.toml

```diff
@@ -0,0 +1,8 @@
+version = 1
+
+[[analyzers]]
+name = "go"
+enabled = true
+
+  [analyzers.meta]
+  import_root = "github.com/uzaxirr/zestream-server"
```

Comment:
will it be zestream or uzaxirr in the url?

Reply:
This file got added up by mistake, will revert this